static/templates/settings/realm-filter-settings-admin.handlebars

```diff
@@ -1,7 +1,7 @@
 <div id="filter-settings" class="settings-section" data-name="filter-settings">
   <div class="settings-section-title"><i class="icon-vector-filter settings-section-icon"></i>{{t "Custom linkification filters" }}</div>
   <div class="admin-table-wrapper">
-    <p>{{#tr this}}Configure patterns that will be automatically linkified when used in Zulip message bodies or topics.  For example, you could make typing "#123" automatically become a link to issue #123 in an issue tracker.{{/tr}}</p>
+    <p>{{#tr this}}Configure patterns that will be automatically linkified when used in Zulip message bodies or topics.  For example, "#123" can be automatically linkified to issue #123 in zulip/zulip repository by adding "#(?P&lt;id&gt;[0-9]+)" as regular expression and "https://github.com/zulip/zulip/issues/%(id)s" as URL format string.  Similarly, adding "(?P&lt;id&gt;[0-9a-f]{32})" as regular expression and "https://github.com/zulip/zulip/commit/%(id)s" as URL format string automatically linkifies all the commit IDs to corresponding commits in zulip/zulip repository.{{/tr}}</p>
```

Comment:
This isn't markdown -- it's HTML.  Oh, yeah, so we'll need to use explicit `<ul>` and `<li>` HTML tags for the bullets.